doc/templates/index.html

```diff
@@ -33,233 +35,274 @@ <h4 class="sk-landing-subheader text-white font-italic mb-3">Machine Learning in
+        </p>
       </div>
+      <!--Testimonials -->
       <div class="col-md-4">
         <h4 class="sk-landing-call-header">Who uses scikit-learn?</h4>
-        <div id="carouselExampleSlidesOnly" class="carousel slide" data-ride="carousel">
-        <div class="carousel-inner">
+        <div id="skWhoUsesCarousel" class="carousel slide sk-who-uses-carousel" data-bs-ride="carousel" data-bs-interval="5000">
+          <div class="carousel-inner">
             <div class="carousel-item active">
-            <img class="d-block mx-auto sk-who-uses-carousel-img img-thumbnail" src="_images/inria.png" alt="inria">
-            <em>"We use scikit-learn to support leading-edge basic research [...]"</em>
+              <img class="d-block mx-auto img-thumbnail" src="_images/inria.png" alt="inria">
+              <em>"We use scikit-learn to support leading-edge basic research [...]"</em>
             </div>
             <div class="carousel-item">
-            <img class="d-block mx-auto sk-who-uses-carousel-img img-thumbnail" src="_images/spotify.png" alt="spotify">
-            <em>"I think it's the most well-designed ML package I've seen so far."</em>
+              <img class="d-block mx-auto img-thumbnail" src="_images/spotify.png" alt="spotify">
+              <em>"I think it's the most well-designed ML package I've seen so far."</em>
             </div>
             <div class="carousel-item">
-            <img class="d-block mx-auto sk-who-uses-carousel-img img-thumbnail" src="_images/change-logo.png" alt="change-logo">
-            <em>"scikit-learn's ease-of-use, performance and overall variety of algorithms implemented has proved invaluable [...]."</em>
+              <img class="d-block mx-auto img-thumbnail" src="_images/change-logo.png" alt="change-logo">
+              <em>"scikit-learn's ease-of-use, performance and overall variety of algorithms implemented has proved invaluable [...]"</em>
             </div>
             <div class="carousel-item">
-            <img class="d-block mx-auto sk-who-uses-carousel-img img-thumbnail" src="_images/telecomparistech.jpg" alt="telecomparistech">
-            <em>"The great benefit of scikit-learn is its fast learning curve [...]"</em>
+              <img class="d-block mx-auto img-thumbnail" src="_images/telecomparistech.jpg" alt="telecomparistech">
+              <em>"The great benefit of scikit-learn is its fast learning curve [...]"</em>
             </div>
             <div class="carousel-item">
-            <img class="d-block mx-auto sk-who-uses-carousel-img img-thumbnail" src="_images/aweber.png" alt="aweber">
-            <em>"It allows us to do AWesome stuff we would not otherwise accomplish"</em>
+              <img class="d-block mx-auto img-thumbnail" src="_images/aweber.png" alt="aweber">
+              <em>"It allows us to do Awesome stuff we would not otherwise accomplish."</em>
```

Comment:
I think the casing was intentional (AWeber):

```suggestion
              <em>"It allows us to do AWesome stuff we would not otherwise accomplish."</em>
```